crest/Assets/Crest/Crest/Scripts/LodData/RegisterClipSurfaceInput.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Crest Ocean System
+
+// This file is subject to the MIT License as seen in the root of this folder structure (LICENSE)
+
+using UnityEngine;
+
+namespace Crest
+{
+    /// <summary>
+    /// Tags this object as a clip surface provider. Renders depth every frame and should only be used for dynamic objects.
+    /// </summary>
+    public class RegisterClipSurfaceInput : RegisterLodDataInput<LodDataMgrClipSurface>
+    {
+        [SerializeField] bool _assignClipSurfaceMaterial = true;
+
+        public override float Wavelength => 0f;
+
+        protected override Color GizmoColor => new Color(1f, 0f, 0f, 0.5f);
```

Comment:
I guess we should pick some different colour. I'm open to ideas here :). Perhaps black?

Reply:
Black is good